### ISSUE TITLE
Render just stops?

### ISSUE THREAD

**Issue by casperse** [BlueMap-Minecraft/BlueMap#519]
[debug.log](https://github.com/BlueMap-Minecraft/BlueMap/files/14481414/debug.log)
[world.conf.txt](https://github.com/BlueMap-Minecraft/BlueMap/files/14481466/world.conf.txt)
[world_nether.conf.txt](https://github.com/BlueMap-Minecraft/BlueMap/files/14481467/world_nether.conf.txt)
[world_the_end.conf.txt](https://github.com/BlueMap-Minecraft/BlueMap/files/14481468/world_the_end.conf.txt)
[core.conf.txt](https://github.com/BlueMap-Minecraft/BlueMap/files/14481479/core.conf.txt)

### What i did / Steps to reproduce

Install stable version and follow official install guide
Tried both stable and new v4. with same result

### Expected result
A full render over time.

### Actual result
Goes like a rocket and stops after 2min and goes idle?

### Context
BlueMap-3.20-paper.jar
BlueMap-4.0-paper.jar
(Tried both)
 
Add more context here!
- Os and platform (Linux Ubuntu, tried both Spigot & Paper
- Has the world been generated using any Minecraft-mods? No installed except the "blueborder" plugin

Also:
![image](https://github.com/BlueMap-Minecraft/BlueMap/assets/2026859/7a97ec0a-6cd1-46bc-b630-c00ba5e69b94)

Render:
![image](https://github.com/BlueMap-Minecraft/BlueMap/assets/2026859/eea083ea-fc68-47e4-b978-d0cac24a5cfb)

Also some strange errors about the world naming it dosen 't always work?
There is no world with that name? (world, overworld)
Or Can't detect a world from this command-source?

And then after a# bluemap maps or bluemap worlds I can use the command with "world"

-->
Also asked for help here:
https://www.reddit.com/r/BlueMap/comments/1b5lqmn/rendering_stops_and_i_get_renderthreads_are_idle/

Update: Attached debug.log file above

**Comment by casperse** [comment]
So I tried to set the plugin.conf to player-render-limit: 0 (instead of -1) should be the same.
But still goes into idle after a few minutes
![image](https://github.com/BlueMap-Minecraft/BlueMap/assets/2026859/c0b90f08-6e04-47d3-8aac-e261c112035b)

I have 10 cores and it have 8G of RAM and the load is maxing out at 5G.
This is so strange, even forcing a update or deleting the map have no impact.
It runs for 1-2 minutes and then idles

**Comment by TechnicJelle** [comment]
What happens if you, in-game, go to one of the un-rendered areas, and then do a server restart?
Does it appear then?

**Comment by TBlueF** [comment]
BlueMap only renders parts of the world that are already generated by minecraft. So usually only areas that a player has visited ingame. If you want to see more on BlueMap, you'll need to pre-generate the part of your world that you want to see on the map.
You can do this with a pre-generation plugin like e.g. Chunky: https://github.com/pop4959/Chunky/wiki/Installing

If you still experience black areas after that on your map that you wouldn't expect, see: https://bluemap.bluecolored.de/wiki/FAQ.html#q-i-have-a-black-map--a-lot-of-the-map-is-missing

If you need more support with this, let's continue the thread on reddit :)
Or even better on [discord](https://discord.gg/zmkyJa3) where we are the most responsive!

**Comment by casperse** [comment]
Ok just a quick Q.
I have now followed the guide to pre-generate with chunky (I didnt think Bluemap needed this like dynmap).
And after finishing I wanted to do a fullrender with bluemap (Fullrender is not an command option anymore?)
Tried start but it idles again?

![image](https://github.com/BlueMap-Minecraft/BlueMap/assets/2026859/1753e7d7-77b5-4056-9f1b-8139e230707b)

Do I need to do a force-update after the pre-rendering?
I noticed that chunky uses command chunky world "worldname" is there some other command for bluemap?
bluemap world and then the world names defined by bluemap.

**Comment by casperse** [comment]
[dump.json](https://github.com/BlueMap-Minecraft/BlueMap/files/14520794/dump.json)

1. Okay I removed the 4.0 (Deleted all folder for bluemap)
2. Did a prerendering with chunky.
worldborder get
worldborder set 10000
chunky world world <---- name of world
chunky worldborder
chunky shape square - https://github.com/pop4959/Chunky/wiki/Shapes
chunky spawn - (Defines world center to your spawn point https://apexminecrafthosting.com/chunky-plugin/
chunky center (Where your standing)
chunky radius 10000
chunky start

It finished and I wanted to start rendering with bluemap.
I installed bluemap the stable v3.20
Nothing it just idles, on the webserver I can see that it have done some very small rendering of the_nether & the_end.
But nothing on the world?
But status say that it is idle (Like before)
I have uploaded the dump file, will try to ask someone on discord to help troubleshooting.
Updating this for reference

![image](https://github.com/BlueMap-Minecraft/BlueMap/assets/2026859/7895e133-5f99-46a0-acf8-bf6781d7a21d)
